index.bs

```diff
@@ -498,9 +498,17 @@ Let's examine a few examples:
     set, enabling sites to count them.
     - `"Chrome"; v="73", "Awesome Browser"; v="60", "Chromium"; v="73"`
 
-When choosing GREASE strategies, [=user agents=] SHOULD keep caching variance in mind and minimize variance among identical [=user agent=] versions.
+When choosing GREASE strategies, [=user agents=] SHOULD keep caching variance in mind and minimize
+variance among identical [=user agent=] versions.
 
-Note: One approach to minimize caching variance could be to determine the GREASE parts of the UA set at build time, and keep them identical throughout the lifetime of the [=user agent=]'s significant version. 
+Note: One approach to minimize caching variance could be to determine the GREASE parts of the UA set
+at build time, and keep them identical throughout the lifetime of the [=user agent=]'s significant
+version. 
+
+When adding arbitrary values to [=user agent/brands=], [=user agents=] SHOULD make sure that those
+arbitrary values enforce [=Structured Header=] parsing, by adding escaped double-quotes, commas and
```

Comment:
"adhere to"? "enforce" sounds weird.

Reply:
The goal of that is to make sure the servers use standard Structure Header Parsing rather than try to regex their way through those strings.

So the strings need to ensure that the receivers of those strings adheres to SH parsing rules. I clarified that in the text.